src/Plugin/jsonapi/FieldEnhancer/EmbedVideoEnhancer.php

```diff
@@ -147,9 +149,9 @@ public function parseVideos($video = NULL) {
               $video_id = $results[1];
 
               try {
-                $hash = unserialize(file_get_contents("//vimeo.com/api/v2/video/$video_id.php"));
+                $hash = unserialize(file_get_contents($request->getScheme() . "://vimeo.com/api/v2/video/$video_id.php"));
```

Comment:
Hey @enzolutions I get that protocol less URLs are a bit of an anti pattern now, but should we just always get the contents from `https://`?